Runtime/Scripts/GltfImport.cs

```diff
@@ -49,6 +49,10 @@ namespace GLTFast {
     using Loading;
 
     public class GltfImport : IGltfReadable, IGltfBuffers {
+    
+    
```

Comment:
Two concerns:
1. I'm not sure Actions/Events are the best approach to inject custom loading behavior. Up until now I encouraged people to implement custom IInstantiators (or derive from GameObjectInstantiator) to achieve this. Granted, `IGltfReadable` and `IInstantiator` need to be extended to get access to the `extras`, but that's probably cleaner.
2. This creates a dependency to GameObjects and GameObjectInstantiator. It would not work with a DOTS workflow. This is a blocker.